source/code/plugins/VMInsightsDataCollector.rb

```diff
@@ -54,23 +54,28 @@ def get_available_memory_kb
             return available, total
         end
 
-        # returns: cummulative uptime, cummulative idle time
+        # returns: cummulative total time, cummulative idle time
+
+        # /proc/stat contains system statistics since last restart
+        # first line contains aggregate across all CPUs
+        # format:
+        #
+        # cpu user nice system idle iowait irq softirq steal guest guest_nice
+        # eg: cpu  2904083 315778 1613190 140077550 216726 0 88355 0 0 0
+        # https://www.kernel.org/doc/html/latest/filesystems/proc.html#miscellaneous-kernel-statistics-in-proc-stat
         def get_cpu_idle
-            uptime = nil
+            total = nil
             idle = nil
-            File.open(File.join(@root, "proc", "uptime"), "rb") { |f|
+            File.open(File.join(@root, "proc", "stat"), "rb") { |f|
                 line = f.gets
-                next if line.nil?
-                line.scanf(" %f %f ") { |u, i|
-                    uptime = u
-                    idle = i
+                raise Unavailable, "/proc/stat empty" if line.nil?
+                a = line.split(" ")
```

Comment:
"a" doesn't seem an intuitive name for a variable. Can you change to something meaningful?